src/components/MovieCard/index.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  WrapItem,
+  Link,
+  Image,
+  Heading,
+  Text,
+  List,
+  ListItem,
+} from '@chakra-ui/react';
+import CardImage from '../../assets/card-image.png';
+import StarFull from '../../assets/starFull.png';
+import StarEmpty from '../../assets/starEmpty.png';
+export const MovieCard = (): JSX.Element => {
+  return (
+    <WrapItem
+      bg="white"
+      p="2.4rem"
+      maxW="343px"
+      borderTopLeftRadius="10rem"
+      borderTopRightRadius="1.6rem"
+      borderBottomLeftRadius="1.6rem"
+      borderBottomRightRadius="10rem"
+    >
+      <Link display="block" _hover={{ textDecor: 'none' }}>
+        <Image src={CardImage} maxW="295px" />
```

Comment:
it's a best practice to use `alt` properties on `<img/>`

Reply:
this is ok!